### PR TITLE
*: Support PyTorch

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

This PR adds PyTorch support

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Ref #26 

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by gaocegege** [comment]
PTAL @ddysher , it blocks incoming PRs.

File: s2i/pytorch-s2i/s2i/bin/assemble [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
```
[the remark this answers is not in the document]
Comment (gaocegege) [reply s2i/pytorch-s2i/s2i/bin/assemble]: This is what the buider image need to implement s2i functionalities. The bin will be packaged into builder image. And the scripts are hand-written by me. Thus we cannot remove them.

[the remark this answers is not in the document]
Comment (ddysher) [reply s2i/pytorch-s2i/s2i/bin/assemble]: I mean this binary `s2i/pytorch-s2i/kubeflow-kernel`, this is also required?

[the remark this answers is not in the document]
Comment (gaocegege) [reply s2i/pytorch-s2i/s2i/bin/assemble]: Oh It is a mistake, I removed it now.

[the remark this answers is not in the document]
Comment (gaocegege) [reply s2i/pytorch-s2i/s2i/bin/assemble]: PTAL

[the remark this answers is not in the document]
Comment (ddysher) [reply s2i/pytorch-s2i/s2i/bin/assemble]: cool, thx

I cannot comment on that binary  :laughing:, sorry for the confusion.

**Pull request merged**